app/config/config.yml

```diff
@@ -199,3 +199,42 @@ scheb_two_factor:
 kphoen_rulerz:
     executors:
         doctrine: true
+
+fos_elastica:
+    clients:
+        default: { host: localhost, port: 9200 }
```

Comment:
this config must be in parameters.yml, I think (and I forgot to do the same for RabbitMQ PR I think)